test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -186,6 +186,18 @@ Describe "Get-ChildItem" -Tags "CI" {
             $app.Target | Should -Not -Be $app.FullName
             $app.LinkType | Should -BeExactly 'AppExeCLink'
         }
+
+        It "Wildcard matching behavior is the same between -Path and -Include parameters when using escape characters" {
+            try {
+                New-Item -Type File 'a`[b]' -ErrorAction SilentlyContinue > $null
+                $WithInclude = Get-ChildItem * -Include 'a```[b`]'
+                $WithPath = Get-ChildItem -Path 'a```[b`]'
```

Comment:
I suggest adding another test where escaped and `[` and `]` are _combined with `*` or `?`_ - because a bug currently lurks there (you inexplicably need an extra round of escaping then); e.g., `Get-Item -Path 'a```[b*'` _breaks_ at the moment.

Reply:
@mklement0 Thanks!

Done.

@adityapatwardhan I don't use `-Cases`so as not to add a new Describe.

Reply:
I removed the test because it seems it looks like a bug. I hope we will discuss this in new issue.